types/chrome/index.d.ts

```diff
@@ -9444,7 +9452,7 @@ declare namespace chrome.tabs {
  * Permissions:  "tabGroups"
  * @since Chrome 89. Manifest V3 and above.
  */
- declare namespace chrome.tabGroups {
+declare namespace chrome.tabGroups {
```

Comment:
Undo that change as well
```suggestion
 declare namespace chrome.tabGroups {
```